tests/Knp/Snappy/AbstractGeneratorTest.php

```diff
@@ -5,13 +5,24 @@
 use Knp\Snappy\AbstractGenerator;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Process\Process;
 use InvalidArgumentException;
 use RuntimeException;
 use ReflectionProperty;
 use ReflectionMethod;
 
 class AbstractGeneratorTest extends TestCase
 {
+    /**
+     * @var string
+     */
+    private static $commandPartDelimiter;
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$commandPartDelimiter = '\\' !== \DIRECTORY_SEPARATOR ? "'" : ''; // command parts which are not quoted on Windows are enclosed by single quotes on Linux
+    }
+
```

Comment:
Without a doubt, it would be nice to get rid of this stuff again. Let me know if you have an idea. 😏

Reply:
I agree but, at the moment, I can't think about a better solution.
In my opinion we can just leave a comment in case someone comes up with a better idea.

What do you think?

```suggestion
    /**
     * @var string
     */
    private static $commandPartDelimiter;

    public static function setUpBeforeClass(): void
    {
        /**
         * Command parts which are not quoted on Windows are enclosed by single quotes on Linux.
         * 
         * We should find a better solution to handle this case. Help is appreciated.
         */
        self::$commandPartDelimiter = '\\' !== \DIRECTORY_SEPARATOR ? "'" : '';
    }

```

Reply:
Sure, fine for me. Should the comment inside `setUpBeforeClass` start with just `/*` because it's not PHPDoc?

Reply:
We can keep it this way IMHO

Reply:
However, [php-cs-fixer complained](https://travis-ci.org/github/KnpLabs/snappy/jobs/726712292#L766) about `/**` so I had to change this to `/*`. 😆